### PR TITLE
Added option to exclude stations from radius

### PR DESCRIPTION
**Pull request by TheDuffman85**
Added option excludeStationIds to exclude stationIds from radius search result. This is useful e.g. if you got a non public or truck exclusive station in the radius.

### CODE REVIEW

**Comment by TheDuffman85** [comment]
There seems to be a deprecated ESLint rule. But this is unrelated to my change.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # MMM-Fuel Changelog
 
+## [2.3.1]
+
+### Added
+
+* Config option `excludeStationIds` to exclude gas stations from radius (Tankerkönig only)
```
Comment (fewieden) [inline CHANGELOG.md:7]: Can you please add this entry to 2.3.0?

That version is still work in progress and unreleased.

File: apis/tankerkoenig.js [hunk 2]
```diff
@@ -106,8 +106,10 @@ function sortByDistance(a, b) {
  * @returns {boolean} To keep or filter the station.
  */
 function filterStations(station) {
+    const excludeStations = Array.isArray(config.excludeStationIds);
```
Comment (fewieden) [inline apis/tankerkoenig.js:109]: I would add `excludeStationIds: []` to the module defaults instead, which would also allow to remove one check in the if statement below

**Review by fewieden — CHANGES_REQUESTED** [review]
@TheDuffman85 thanks for your contribution

**Comment by fewieden** [comment]
> There seems to be a deprecated ESLint rule. But this is unrelated to my change.

It should be fixed in the current develop branch. I committed the fix 21 days ago but only pushed it to github 2 days ago.

https://github.com/fewieden/MMM-Fuel/commit/c15455bb39c27f44bab87eab0d07235a570d03db

File: CHANGELOG.md (hunk 1 above)
Comment (TheDuffman85) [reply CHANGELOG.md:7]: Sure thing.

File: apis/tankerkoenig.js (hunk 2 above)
Comment (TheDuffman85) [reply apis/tankerkoenig.js:109]: I guess this then should also be true for stationIds, right?

**Comment by TheDuffman85** [comment]
> > There seems to be a deprecated ESLint rule. But this is unrelated to my change.
> 
> It should be fixed in the current develop branch. I committed the fix 21 days ago but only pushed it to github 2 days ago.
> 
> [c15455b](https://github.com/fewieden/MMM-Fuel/commit/c15455bb39c27f44bab87eab0d07235a570d03db)

Still failing after I merged the develop branch

**Comment by TheDuffman85** [comment]
I've fixed the issue with the ESLint rule.

**Pull request merged**